resources/default_template.tex

```diff
@@ -0,0 +1,29 @@
+\documentclass[varwidth=true, crop, border=5pt]{standalone}
+
+% Packages
+\usepackage{amsmath}
+\usepackage{amssymb}
+
+% Blank formula checking
+\usepackage{ifthen}
+\newlength{\pheight}
+
+% Color support
+\usepackage{xcolor}
+\definecolor{xpp_font_color}{HTML}{%%XPP_TEXT_COLOR%%}
+
+% User input
+\def\preview{\(
+  \displaystyle
+  %%XPP_TOOL_INPUT%%
+  \)
+}
```

Comment:
This will not be copied verbatim, which is a problem if a user wants to use e.g. code highlighting via the listings package or file contents inclusion via the filecontents package. It's better to use the scontents package to copy the user input verbatim.